src/main/java/software/amazon/awssdk/s3accessgrants/cache/S3AccessGrantsCachedBucketRegionResolver.java

```diff
@@ -0,0 +1,170 @@
+package software.amazon.awssdk.s3accessgrants.cache;
```

Comment:
Missing license file header

Reply:
Seems like the license file was not added automatically. Thanks for catching this. Will add the license file.